go/pkg/zen/auth.go

```diff
@@ -37,17 +40,15 @@ func Bearer(s *Session) (string, error) {
 	}
 
 	header = strings.TrimSpace(header)
-	if !strings.HasPrefix(header, "Bearer ") {
+	if !strings.HasPrefix(strings.ToLower(header), bearerPrefix) {
```

Comment:
why?
what was broken before?

Reply:
It was case sensitive which shouldn't be

Reply:
we're not doing oath, but the bearer auth scheme clearly states it must be uppercase

> All challenges defined by this specification MUST use the auth-scheme
   value "Bearer"

Reply:
Hmm, but doesn't that contradict with headers should be case insensitive? If thats not the case I can revert those